travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
```

Comment:
i cannot enable this on the reposity level, but when enabled this pr should already be run with ci.